linkerd/docs/response_classifier.md

```diff
@@ -80,7 +80,31 @@ All responses are considered to be successful, regardless of status code.
 
 # gRPC Response Classifiers
 
-For HTTP/2 routers that handle gRPC traffic, four additional response classifiers are available to categorize responses based on [gRPC status codes](https://github.com/grpc/grpc/blob/master/doc/statuscodes.md) in the stream's trailers frame. Status code 0 (`OK`) is always considered successful, while all other gRPC status codes are considered failures.
+For HTTP/2 routers that handle gRPC traffic, one optional parameter and four
+additional response classifiers are available to categorize responses based on
+[gRPC status codes](https://github.com/grpc/grpc/blob/master/doc/statuscodes.md)
+in the stream's trailers frame.
+
+By default, status code 0 (`OK`) is always considered successful, while all
+other gRPC status codes are considered failures. There exists an optional
+parameter `nonAccruableStatusCodes` that accepts a user-defined list of error
```

Comment:
This is a property of response classification, so it doesn't just apply to failure accrual, it also applies to metrics, for example.  Therefore, I wouldn't name this `nonAccruableStatusCodes` but would instead name it something like `successStatusCodes`.

Reply:
Okay that makes sense. I'll change the name.